src/demo_impls.rs

```diff
@@ -192,6 +204,25 @@ impl GenerateVerifiable for Simple {
 	fn internal_member(value: &Self::Member) -> Self::InternalMember {
 		value.clone()
 	}
+
+	fn sign(secret: &Self::Secret, message: &[u8]) -> Result<Self::Signature, ()> {
+		let secret = MiniSecretKey::from_bytes(&secret[..]).unwrap();
+		let pair = secret.expand_to_keypair(ExpansionMode::Ed25519);
+
+		let sig = ("no-ctxt", message)
```

Comment:
here I am not so sure, I use `"no-ctxt"` to avoid conflicting with a context used in a proof.

But I think removing it can be fine, maybe if somebody ask me to sign a message and somehow the first bytes of the message is matching a context used somewhere, then it can leak the link between my public key and my alias I guess.

Reply:
This is not a production impl so I think we're ok.